src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -543,7 +541,7 @@ export function confirmCancelAppointment() {
           ? 'cc'
           : 'va',
     };
-    let apiData = appointment.legacyVAR?.apiData || appointment.apiData;
+    let apiData = appointment.vaos.apiData || appointment.apiData;
```

Comment:
Moved `apiData` to the `vaos` sub-object, since that's where we store extra info for the Appointment